faq.md

```diff
@@ -126,4 +126,13 @@ Distributed writes to netCDF are hard.
 
 1. Try writing to `zarr` using `Dataset.to_zarr`.
 1. If you need to write to netCDF and your final dataset can fit in memory then use `dataset.load().to_netcdf(...)`.
-1. If you really must write a big dataset to netCDF try using `save_mfdataset` (see [here](https://ncar.github.io/xdev/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask/)).
+1. If you really must write a big dataset to netCDF try using `save_mfdataset` (see [here](https://ncar.github.io/xdev/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask/)). 
+
+### My Dask workers are taking a long time to start.   How can I monitor them?
+
+Dask worker requests are added to the job queues on Casper and Cheyenne with the `cluster.scale()` method.   After this method is called, you can verify that they are waiting in the queue with these commands:
+*  `squeue --me` on Casper
```

Comment:
Slurm on Casper is being phased out, and I think Casper will exclusively use PBS by mid April.

Reply:
OK, I will change the text to reflect that.   Thanks for the reminder.